mbed_host_tests/host_tests_conn_proxy/conn_primitive_remote.py

```diff
@@ -32,6 +31,12 @@ def __init__(self, name, config):
         self.baudrate = config.get('baudrate', DEFAULT_BAUD_RATE)
         self.image_path = config.get('image_path', None)
         self.polling_timeout = int(config.get('polling_timeout', 60))
+        self.tags = config.get('tags', None)
+
+        if(self.tags):
+            self.tags = {}
+            for tag in config.get('tags', None).split(','):
+                self.tags[tag] = True
```

Comment:
I left this as True for simplicity as it should be the only valid case for this situation (please correct me if I'm wrong).